bitnami/kafka/values-production.yaml

```diff
@@ -385,19 +385,14 @@ service:
   ## Kafka port
   ##
   port: 9092
-  ## Kafka SSL port
-  ##
-  sslPort: 9093
-  ## Specify the nodePort(s) value for the LoadBalancer and NodePort service types.
+  ## Specify the nodePort value for the NodePort service type.
   ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
   ##
-  nodePorts:
-    kafka: ""
-    ssl: ""
+  nodePort:
   ## Set the LoadBalancer service type to internal only.
   ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
   ##
-  # loadBalancerIP:
+  loadBalancerIP:
```

Comment:
Please revert this change too. We faced some issues in the past with parameters with no value associated.
To avoid problems, it's better to have them commented.